src/tools/perf/ucp_tests.cc

```diff
@@ -378,6 +373,16 @@ class ucp_perf_test_runner {
     }
 
 private:
+    void UCS_F_ALWAYS_INLINE send_started()
+    {
```

Comment:
Coding style

Reply:
what is the problem here? AFAIR we have not defined that C++ function block should begin on the same line... also other functions in this file look the same way

Reply:
Ok